src/Link.js

```diff
@@ -1,10 +1,12 @@
 import styled, { css } from 'react-emotion';
 import PropTypes from 'prop-types';
+import { color } from 'styled-system';
 
 import Text from './Text';
 
 const baseStyles = css`
   text-decoration: none;
+  cursor: pointer;
```

Comment:
Is this not the browser default?

Reply:
Yep, you're right. For some reason I thought it broke somewhere - but I just checked it and everything checks out. Not sure what I was thinking here... 🤔